components/stepper/stories/template.js

```diff
@@ -10,96 +12,114 @@ import { when } from "lit/directives/when.js";
 import "../index.css";
 
 export const Template = ({
-	rootClass = "spectrum-Stepper",
+	rootClass = "spectrum-NumberField",
 	size = "m",
-	isQuiet = false,
+	displayLabel = true,
+	label,
+	labelPosition = "top",
 	isFocused = false,
 	isHovered = false,
 	isKeyboardFocused = false,
 	isInvalid = false,
 	isDisabled = false,
 	hideStepper = false,
-	id = getRandomId("stepper"),
+	helpText,
+	value = "0",
+	id = getRandomId("numberfield"),
 	customClasses = [],
 	customStyles = {},
 } = {}, context = {}) => {
-	let iconSize = "75";
-	switch (size) {
-		case "s":
-			iconSize = "50";
-			break;
-		case "l":
-			iconSize = "100";
-			break;
-		case "xl":
-			iconSize = "200";
-			break;
-		default:
-			iconSize = "75";
-	}
-
+	const { updateArgs } = context;
 	return html`
 		<div
 			class=${classMap({
 				[rootClass]: true,
-				[`${rootClass}--size${size?.toUpperCase()}`]: typeof size !== "undefined",
-				[`${rootClass}--quiet`]: isQuiet,
+				[`${rootClass}--size${size?.toUpperCase()}`]: typeof size !== "undefined" && size !== "m",
+				[`${rootClass}--sideLabel`]: labelPosition === "side",
+				[`${rootClass}--hiddenStepper`]: hideStepper,
 				"is-focused": isFocused,
 				"is-hover": isHovered,
 				"is-keyboardFocused": isKeyboardFocused,
 				"is-invalid": isInvalid,
 				"is-disabled": isDisabled,
-				"hide-stepper": hideStepper,
 				...customClasses.reduce((a, c) => ({ ...a, [c]: true }), {}),
 			})}
+			@keyup=${function(e) {
```

Comment:
Inspiration from this PR: https://github.com/adobe/spectrum-css/pull/3354

I wanted to make sure the correct keyboard focus styles were being applied when I actually navigated to the number field with the tab key.